app/src/test/java/org/mozilla/fenix/home/intent/DeepLinkIntentProcessorTest.kt

```diff
@@ -244,9 +249,31 @@ class DeepLinkIntentProcessorTest {
     }
 
     @Test
-    fun `process make_default_browser deep link`() {
+    @Config(minSdk = N, maxSdk = P)
```

Comment:
Nice!
Probably we don't need to hardcode the maxSdk though.

Reply:
Removed as suggested.

Reply:
I tried removing the `maxSdk = P` and I am getting this error on the build agent: 
`java.lang.UnsupportedOperationException: Failed to create a Robolectric sandbox: Android SDK 29 requires Java 9 (have Java 8)`

I reverted my change in order to get the build working, looks like a Roboelectric issue and I am unsure if I should try including a fix here.